app/uk/gov/hmrc/tariffclassificationfrontend/connector/CasesConnector.scala

```diff
@@ -30,6 +30,11 @@ import scala.concurrent.Future
 @Singleton
 class CasesConnector @Inject()(configuration: AppConfig, client: HttpClient) {
 
+  def getOne(reference: String)(implicit hc: HeaderCarrier): Future[Option[Case]] = {
+    val url = configuration.bindingTariffClassificationUrl + "/cases/" + reference
```

Comment:
I don't know if we are going to call more services, but I would prefer to create a common structure for all of them
myService:
    protocol: https
    domain: myService.qa.gov.uk
    port: 1111
    timeout: 5s
    ...

This is the minimum I would set on a config file. we could set as well the endpoints but this point could be discussed

Reply:
Have a look at the `microservice.service` block in `application.conf`. This gets picked up by `AppConfig` (see line 38).